ops/cloudformation/vpc.yml

```
@@ -0,0 +1,239 @@
+---
+  AWSTemplateFormatVersion: "2010-09-09"
+
+  Description: "Creates a VPC with Managed NAT"
+
+  Parameters:
+    Environment:
+      Type: String
+      AllowedValues: [ Dev, Prod ]
+      ConstraintDescription: Must be a valid environment name (Dev or Prod)
+
+  Mappings:
+    SubnetConfig:
+      VPC:
+        Dev: "172.16.0.0/16"
+        Prod: "10.0.0.0/16"
+      Public0:
+        Dev: "172.16.0.0/24"
+        Prod: "10.0.0.0/24"
+      Private0:
+        Dev: "172.16.1.0/24"
+        Prod: "10.0.1.0/24"
+      Private1:
+        Dev: "172.16.2.0/24"
+        Prod: "10.0.2.0/24"
+
+    AZRegions:
```

as we have only one public subnet, i would change mapping to avoid confusion to something like:
```suggestion
    AZRegions:
      us-east-1:
        publicAZs: ["a"]
        privateAZs: ["a", "b"]
      us-west-1:
        publicAZs: ["a"]
        privateAZs: ["a", "b"]
```

---

well, it makes sense